test/blackbox/common/BlackboxTestsTransportSHM.cpp

```diff
@@ -114,12 +114,18 @@ TEST(SHM, IgnoreNonExistentSegment)
     reader.wait_discovery();
 
     // Create and quickly destroy several participants in several threads
+#ifdef _WIN32
+    constexpr size_t num_threads = 1;
```

Comment:
Please use this for the loop of creating the threads

Reply:
9fedf5819718addef06383ea396ad1506f3b9f3a